lib/wunderlist/api.rb

```diff
@@ -199,6 +198,10 @@ def delete(url, options = {})
 
 
     def get_list_ids(list_names = [])
+      return list_names if list_names.all? {|i| i.is_a?(Integer) }
```

Comment:
If all elements in the array are integer, we suppose they are ID and return them